tests/ApiTest.php

```diff
@@ -31,8 +88,39 @@ public function testGet()
     public function testCreate()
     {
         $client = $this->getClient(self::objectClass);
+
+        $mockBody = '{
+  "meta": {
+    "version": 1,
+    "status_code": 200
+  },
+  "results": {
+    "protocol": "https",
+    "apiFormat": "json",
+    "endPointTimeout": 2,
+    "disabled": false,
+    "strictSSL": true,
+    "sendThroughApiKey": false,
+    "sendThroughApiSig": false,
+    "endPoint": "myapiendpoint.com",
+    "createdAt": 1389915291013,
+    "tokenSkewProtectionCount": 3,
+    "hasCapturePaths": false,
+    "allowKeylessUse": false,
+    "keylessQps": 2,
+    "keylessQpd": 172800,
+    "updatedAt": 1427892638934
+  }
+}';
+
+        $wantBody = '{"protocol":"https","tokenSkewProtectionCount":3,"apiFormat":"json",' .
+            '"endPointTimeout":2,"defaultPath":"\/api","disabled":false,"strictSSL":true,' .
+            '"sendThroughApiKey":false,"sendThroughApiSig":false,"hasCapturePaths":false,' .
+            '"allowKeylessUse":false,"keylessQps":2,"keylessQpd":172800}';
```

Comment:
I don't know whether you hand-coded these values or they were templates/examples to use for this, but if you want to make these more readable, you could code them as PHP arrays and `json_encode()` them:
```suggestion
        $wantBody = json_encode([
            'protocol' => 'https',
            'tokenSkewProtectionCount' => 3,
            'apiFormat' => 'json',
            'endPointTimeout' => 2,
            'defaultPath' => '/api',
            'disabled' => false,
            'strictSSL' => true,
            'sendThroughApiKey' => false,
            'sendThroughApiSig' => false,
            'hasCapturePaths' => false,
            'allowKeylessUse' => false,
            'keylessQps' => 2,
            'keylessQpd' => 172800,
        ]);
```

Just a suggestion.

Reply:
Done